src/lua/dojitcmd.lua

```diff
@@ -0,0 +1,39 @@
+local M = {}
+
+local function runcmdopt(func, opt)
+    if opt ~= nil and string.sub(opt, 1, 1) ~= "" then
+        local args = opt:split(',')
+        for i = 1, #args do
+            if args[i] == "" then args[i] = nil end
+        end
+        return pcall(func, unpack(args))
```

Comment:
```lua
tarantool> unpack({1, nil, 3})
---
- 1
...
```

`unpack()` has no guarantees regarding holed arrays. In such a case it is better to use `unpack(args, 1, table.maxn(args))`.

BTW, what this `nil` assignment logic is doing? Shouldn't it be tested?

Reply:
Yeah, agreed that these assignments to nil looks confusing.

Why do not simple pass to `return pcall(func, unpack(args))` without that massaging of `args`?

Reply:
Hm. Things a bit strange here for me, but okay, the original question is irrelevant now and I'll resolve this discussion.